Update network interaction related BBEs (#2921)
ballerina-platform/ballerina-distribution: examples/resource-methods/resource_methods.md

# Resource methods

> Service objects use `remote` methods to expose services in procedural style. Remote methods are named by verbs.

Reviewer: ```suggestion
Service objects use `remote` methods to expose services in a procedural style. Remote methods are named by verbs.
```

> Service objects use `resource` methods to expose services in an RESTful style. Resources are named by nouns. Resources are motivated by HTTP, but are general enough also to work for GraphQL. `resource` methods are a network-oriented generalization of OO getter/setter concept.

Reviewer: ```suggestion
Service objects use `resource` methods to expose services in a RESTful style. Resources are named by nouns. Resources are motivated by HTTP but are general enough to work for GraphQL. `resource` methods are a network-oriented generalization of the OO `getter/setter` concept.
```
Better to say what is `OO` here and have the abbreviation in front within brackets.
